website/src/views/planner/ModuleMenu.tsx

```diff
@@ -17,13 +17,44 @@ type MenuItem = {
 };
 
 const ModuleMenu = memo((props: Props) => {
+  const [isVisible, setIsVisible] = useState<boolean>(true);
```

Comment:
visible is a bit awkward since the menu isn't visible when it is closed either 

```suggestion
  const [isMenuOverflowing, setMenuOverflowing] = useState(true);
```

Reply:
:+1: